lib/tests/test_local_working_copy.rs

```diff
@@ -812,7 +811,7 @@ fn test_checkout_discard() {
         store.clone(),
         workspace_root.clone(),
         state_path.clone(),
-        &user_settings(),
+        repo.settings(),
```

Comment:
nit: should be squashed to the first commit

Reply:
Done.